### ISSUE TITLE
TypeError on Windows

### ISSUE THREAD

**Issue by ryanplant-au** [AtomLinter/linter-elixirc#53]
Atom 1.5.3 with linter-elixirc 1.1.5 and linter 1.11.3 on Windows 10 reliably produces this error whenever linting is triggered:

```text
TypeError: Path must be a string. Received null
    at assertPath (path.js:8:11)
    at Object.win32.relative (path.js:256:3)
    at isTestFile (file:///C:/Users/Ryan/.atom/packages/linter-elixirc/lib/init.coffee:47:27)
    at getDepsPa (file:///C:/Users/Ryan/.atom/packages/linter-elixirc/lib/init.coffee:107:16)
    at file:///C:/Users/Ryan/.atom/packages/linter-elixirc/lib/init.coffee:118:7
    at Object.provider.lint (file:///C:/Users/Ryan/.atom/packages/linter-elixirc/lib/init.coffee:133:11)
    at promises.push.Promise.then._this.emitter.emit.linter (C:/Users/Ryan/.atom/packages/linter/lib/linter-registry.js:63:26)
    at C:/Users/Ryan/.atom/packages/linter/lib/linter-registry.js:62:23
    at Set.forEach (native)
    at LinterRegistry.lint (C:/Users/Ryan/.atom/packages/linter/lib/linter-registry.js:60:18)
    at file:///C:/Users/Ryan/.atom/packages/linter/lib/linter.coffee:98:16
    at Function.module.exports.Emitter.simpleDispatch (C:\Users\Ryan\AppData\Local\atom\app-1.5.3\resources\app.asar\node_modules\event-kit\lib\emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (C:\Users\Ryan\AppData\Local\atom\app-1.5.3\resources\app.asar\node_modules\event-kit\lib\emitter.js:125:28)
    at C:/Users/Ryan/.atom/packages/linter/lib/editor-linter.js:30:20
    at Function.module.exports.Emitter.simpleDispatch (C:\Users\Ryan\AppData\Local\atom\app-1.5.3\resources\app.asar\node_modules\event-kit\lib\emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (C:\Users\Ryan\AppData\Local\atom\app-1.5.3\resources\app.asar\node_modules\event-kit\lib\emitter.js:125:28)
    at TextBuffer.module.exports.TextBuffer.saveAs (C:\Users\Ryan\AppData\Local\atom\app-1.5.3\resources\app.asar\node_modules\text-buffer\lib\text-buffer.js:1067:27)
    at TextBuffer.module.exports.TextBuffer.save (C:\Users\Ryan\AppData\Local\atom\app-1.5.3\resources\app.asar\node_modules\text-buffer\lib\text-buffer.js:1037:19)
    at TextEditor.module.exports.TextEditor.save (C:\Users\Ryan\AppData\Local\atom\app-1.5.3\resources\app.asar\src\text-editor.js:630:26)
    at Pane.module.exports.Pane.saveItem (C:\Users\Ryan\AppData\Local\atom\app-1.5.3\resources\app.asar\src\pane.js:545:18)
    at Pane.module.exports.Pane.saveActiveItem (C:\Users\Ryan\AppData\Local\atom\app-1.5.3\resources\app.asar\src\pane.js:528:19)
    at Workspace.module.exports.Workspace.saveActivePaneItem (C:\Users\Ryan\AppData\Local\atom\app-1.5.3\resources\app.asar\src\workspace.js:649:35)
    at atom-workspace.commandRegistry.add.core:save (C:\Users\Ryan\AppData\Local\atom\app-1.5.3\resources\app.asar\src\register-default-commands.js:173:32)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (C:\Users\Ryan\AppData\Local\atom\app-1.5.3\resources\app.asar\src\command-registry.js:260:29)
    at C:\Users\Ryan\AppData\Local\atom\app-1.5.3\resources\app.asar\src\command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (C:\Users\Ryan\AppData\Local\atom\app-1.5.3\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:536:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (C:\Users\Ryan\AppData\Local\atom\app-1.5.3\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:359:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeydown (C:\Users\Ryan\AppData\Local\atom\app-1.5.3\resources\app.asar\src\window-event-handler.js:97:36)
    at HTMLDocument.<anonymous> (C:\Users\Ryan\AppData\Local\atom\app-1.5.3\resources\app.asar\src\window-event-handler.js:3:61)
```

**Comment by Arcanemagus** [comment]
Looks like it's failing to check for a `null` [here](https://github.com/AtomLinter/linter-elixirc/blob/v1.1.5/lib/init.coffee#L47).

@remiq The project path gathering should probably be replaced with a call to `atom.project.relativizePath()`, and the return value should definitely be checked at the above location.

**Comment by Arcanemagus** [comment]
Looks like I had accidentally fixed this already in #45, it was just waiting on testers. Since none ever showed up and it works on every platform I can test I went ahead and merged that in. I'll be pushing out a release incorporating that soon.

**Comment by Arcanemagus** [comment]
@JohnBat26, @ryanplant-au: This should be fixed in v1.1.6, please let us know if you run into any issues with this release!
